tests/python/pants_test/engine/examples/scheduler_inputs/src/thrift/BLD.json

```diff
@@ -43,25 +43,25 @@
 {
   "type_alias": "scrooge_scala_configuration",
   "name": "scrooge_scala_config",
-  "rev": "3.20.0",
+  "rev": "18.4.0",
   "strict": true,
   "dependencies": [
     {
       "type_alias": "jar",
       "extends": ":libthrift",
-      "rev": "0.6.1"
+      "rev": "0.9.2"
     },
     {
       "type_alias": "jar",
       "org": "com.twitter",
       "name": "scrooge-core_2.10",
```

Comment:
We use `2.11` in `BUILD.tools`. Should this also be `2.11`? It's referring to Scala version, right?